tf_encrypted/convert/register.py

```diff
@@ -544,8 +572,10 @@ def inputter_fn():
     return x_public
 
 
-def nodef_to_private_pond(converter, x):
+def nodef_to_private_pond(converter, x, int_warn=False, int_err=False):
```

Comment:
What's the idea behind the `int_warn` and `int_err`? I don't see them being used anywhere.

Reply:
the idea was that most of the time you'll have floats or want to convert to those before defining an input, so if you end up with ints at that point you might have a bug/unexpected behavior.  I think I deleted the code that used it :(

Reply:
Seems reasonable to me!

Reply:
I agree, if it's just for setting the log level then it should probably be done elsewhere